include/constants/pokemon.h

```diff
@@ -63,7 +63,7 @@ enum PokemonDataParam {
     MON_DATA_3,
     MON_DATA_4,
     MON_DATA_SPECIES,
-    MON_DATA_HELD_ITEM,
+    MON_DATA_HELD_ITEM, // 6
```

Comment:
remove these comments (and others)

Reply:
done. I left the note about the data sanity because that might be helpful as people figure out what each piece of data does